src/antq/util/maven.clj

```diff
@@ -79,11 +79,24 @@
      :artifact artifact
      :remote-repos remote-repos}))
 
-(defn ^Model read-pom
+(defn- ^Model read-pom*
   [^String url]
   (with-open [reader (io/reader url)]
     (.read (MavenXpp3Reader.) reader)))
 
+(defn ^Model read-pom
+  [^String url]
+  (loop []
+    (or (try
+          (read-pom* url)
+          (catch java.net.ConnectException e
+            (if (= "Operation timed out" (ex-message e))
+              (log/error (str "Fetching pom from " url " failed because it timed out, retrying"))
```

Comment:
`recur` should be done by here

Reply:
I don't understand this? I check for "Operation timed out", if not, I want to rethrow the exception. And I don't know what tail position I can recur from otherwise?

Reply:
Oh sorry, I forgot that we can't `recur` in `catch`. (leads `Can only recur from tail position` error)
It's ok as it is.